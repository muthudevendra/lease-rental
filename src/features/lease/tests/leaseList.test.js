import React, {useEffect} from "react";
import { Provider } from "react-redux";
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
import configureMockStore from "redux-mock-store";
import Lease from "../leaseList";

Enzyme.configure({ adapter: new Adapter() })

const mockStore = configureMockStore();
const store = mockStore({
    loading: false,
    leaseData: []
});
let wrapper;

describe("page LeastList", () => {
    beforeAll(() => {
        wrapper = shallow(
            <Provider store={store}>
                <Lease />
            </Provider>
        ).dive();
        jest.spyOn(React, 'useEffect').mockImplementation(f => f());
    });

    it("matches to snapshot", () => {
        expect(wrapper).toMatchSnapshot();
    });
});