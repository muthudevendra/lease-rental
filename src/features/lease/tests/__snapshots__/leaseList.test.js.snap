// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page LeastList matches to snapshot 1`] = `ShallowWrapper {}`;
